analyzers/src/SonarAnalyzer.CSharp/Helpers/CountComparisonSyntaxHelper.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace SonarAnalyzer.Helpers
+{
+    public static class CountComparisonSyntaxHelper
```

Comment:
To follow previous comment, this class and file should be
```suggestion
    public static class BinaryExpressionSyntaxExtensions
```